bower.json

```diff
@@ -17,6 +17,6 @@
   ],
   "dependencies": {
     "purescript-prelude": "master",
-    "purescript-type-equality": "master"
+    "purescript-type-equality": "leibniz"
```

Comment:
I’ll point back to master once https://github.com/purescript/purescript-type-equality/pull/11 is merged.